src/main/java/Router.java

```diff
@@ -8,69 +8,62 @@
 
 public class Router implements UI {
 
-    private Board board = new Board();
-    private UI ui = this;
-    private Game game = new Game(ui, board);
-
+    private game.GridForWebConverter converter = new game.GridForWebConverter();
 
     public void run() {
-        game.playerSetUp();
-        get("/", (request, response) -> displayCurrentStateOfGame());
+        get("/", (request, response) -> displayCurrentStateOfGame(new Board()));
         get("/makeMove/:move", (request, response) -> makeMoveAndUpdateBoard(request));
     }
 
-    public String createQueryValueForGridState(List<Mark> grid) {
-        StringBuilder stringBuilder = new StringBuilder();
-        for (int markPosition = 0; markPosition < grid.size(); markPosition++) {
-            if (grid.get(markPosition).equals(Mark.EMPTY)) {
-                stringBuilder.append(String.format("%s", (markPosition + 1)));
-            } else {
-                stringBuilder.append(String.format("%s", grid.get(markPosition).toString()));
-            }
-        }
-        return stringBuilder.toString();
-    }
-
     private String makeMoveAndUpdateBoard(Request request) {
-        String move = request.params("move");
-        WebApplicationPlayer player = (WebApplicationPlayer) game.currentPlayer;
+        String move = request.queryParams("move");
+        String currentBoard = request.queryParams("currentBoard");
+        Board board = new Board(3, converter.convertToGridOfMarks(currentBoard));
```

Comment:
Did you think about hiding the converter within the Board? Meaning, it would be nicer from this class to just pass the string representation to the board constructor instead of needing to instantiate a converter here.